script/Ciel/Day 14/14_01B_01_CIEL14B_1B.txt

```diff
@@ -594,52 +679,56 @@ Hearing the latter dispels the fear that had gripped me.
 [sha:60c63423c15f22d273e4ea53ba6d337ec07dd762]{
 -- Page 73, Offset 38557.
 -- 「先輩……!?」
-"Senpai...?!"
+"Senpai?!"
 }
 [sha:4cfd8205d7b3d0a6edc829cfa48b9bc2d9a01685]{
 -- Page 73, Offset 38558.
 -- 「いいから、早く逃げるんです……！
-"Save it, we have to get out of here fast...!
+"Save it, we have to get out of here right away!
 }
 [sha:3fa26cb07edee496044e7e5c62432835c883d6d3]{
 -- Page 73, Offset 38559.
 -- 彼女の蘇生が終わる前に、早く！」
 Quickly, before she fully resurrects!"
 }
-[sha:fb286836297b86f36a76f3443ba7b535d0f3d63b]{
+[offset:38560]{
 -- Page 73, Offset 38560.
 -- 「あ――――」
-"Ah―――"
+"%{i}Ah%{/i}―――"
 }
 [sha:b63e81c41b08a13f4d38119f27908a3c750fd159]{
 -- Page 74, Offset 38561.
 -- 先輩は当惑する俺の腕を掴むと、一足で上空まで跳び上がった。
-Senpai grabs the arm of my perplexed self and soars to the sky with a single step.
+Senpai grabs my arm while I stand there bewildered,
+ then kicks off and soars into the sky.
 }
 [sha:6b68207b5fab4625d4d0fd45b001f67a144805f0]{
 -- Page 74, Offset 38562.
 -- 一軒家の高さを軽く超え、建物の屋根に着地する。
-She easily passes over the height of a home and lands on the roof of a building.
+She effortlessly leaps higher than a house,
+ and touches down atop the roof.
```

Comment:
"higher than a house" sounds a bit weird.
made me think of weed
>...higher than most houses are tall, and...

Reply:
I just think of the old dumb joke
> I can jump higher than a house
> You can?
> Yeah houses can't jump very high at all

Reply:
🤔 
They haven't seen the houses in F/GO